package.json

```diff
@@ -20,7 +20,6 @@
     "amsterdam-react-final-form": "^0.1.77",
     "amsterdam-scaffold-form": "^0.1.26",
     "axios": "^0.19.2",
-    "axios-cache-adapter": "^2.5.0",
```

Comment:
Decided to ditch `axios-cache-adapter`.
A useReducer is saving the cache now, didn't make sense to add an extra abstraction layer.

Reply:
I think so, yeah. Good one.